frontend/src/pull.ts

```diff
@@ -123,6 +123,7 @@ export class Pull extends PullData {
   isReady(): boolean {
     return (
       this.hasMetDeployRequirements() &&
+      !this.isDraft() &&
```

Comment:
Maybe this should be in the `hasMetDeployRequirements` function?

Reply:
We consider a draft as a type of dev-block so I think it would be best grouped with the other conditions that check for blocks

```
      !this.isDraft() &&
      !this.getDevBlock() &&
      !this.getDeployBlock() &&
```

Reply:
It may be better to group those into their own hasBlock() function